.gitignore

```diff
@@ -22,3 +22,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 .env
+yalc.lock
```

Comment:
what is this 😂

Reply:
Local linking of depedency projects (the UI lib): https://www.npmjs.com/package/yalc